docs/ml-guide.md

```diff
@@ -455,6 +465,14 @@ val pipeline = new Pipeline()
 // Fit the pipeline to training documents.
 val model = pipeline.fit(training)
 
+// now we can optionally save it to disk
+model.save("/tmp/spark-logistic-regression-model")
```

Comment:
One more thought: It'd be cool to also demonstrate saving and loading "pipeline" (the unfit pipeline) above.